src/app/cmd/drop_on_timeline.cpp

```diff
@@ -107,13 +107,13 @@ void DropOnTimeline::onExecute()
       }
 
       // If there is only one source document to process and it has a cel that
-      // can be moved, then move the cel from the source doc into the
+      // can be moved, then copy the cel from the source doc into the
       // destination doc's selected frame.
       const bool isJustOneDoc = (docsProcessed == 1 && !hasPendingWork());
       if (isJustOneDoc && canMoveCelFrom(srcDoc.get())) {
```

Comment:
Probably we have to update the comment that says "it has a cel that can be moved" and the `canMoveCelFrom()` member function name.

Reply:
Sure! I will apply this to https://github.com/aseprite/aseprite/pull/5106 as well, because that PR will actually remove the DropOnTimeline command in favor of using the DocApi class.

Reply:
Hmmm, what if we just left this as is and I just put the changes on my draft? Because this changes will be lost anyway when that PR got merged.

Reply:
This is the commit in the other PR: https://github.com/aseprite/aseprite/pull/5106/commits/59dc18213f1d5d93d11caf1fa9d9add70b1311fd